src/config/i18n/en-us.json

```
@@ -388,5 +388,7 @@
 	"viewMoreStatistics": "View More Statistics",
     "nodeCountChartTitle": "Node count over time",
     "timestamp": "Timestamp",
-    "symbolTime": "Symbol Time"
+    "symbolTime": "Symbol Time",
+    "restVersion": "Rest Version",
```

Is the user familiar with the concept of a REST API? Should we talk about REST at all, or maybe it is enough to say API?

In any case, REST should be capitalized.
```suggestion
    "restVersion": "REST Version",
```

---

I think this is more for the Developer to view.
IMO REST should be fine because the GitHub repo called `catapult-rest`